package.json

```diff
@@ -28,15 +28,13 @@
     "version": "changeset version && pnpm install --no-frozen-lockfile && pnpm run format"
   },
   "workspaces": [
-    "compiled/*",
```

Comment:
Prettier doesn't like that those folders don't exist, it instantly crashes with an error that the folder doesn't exist when plugins are enabled. 

I tried disabling automatic plugin search, but that didn't solve the issue. I saw a few issues on their repo about this (notably a lot of people talking about MongoDB and Docker folders not existing)